packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
+
 # Field
 
-These captions help make forms more accessible by providing context to the user. They can be used with all Odyssey inputs.
+Field provides all of the supporting context required by inputs, including labels, hints, and errors.
+
+This component is also responsible for managing the appropriate application of `aria-` attributes and `id` associations between Inputs and their associated elements.
+
+[View source on GitHub](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/src/components/Field)
 
 ## Anatomy
 
-**STORY MISSING**
+<Canvas>
+  <Story id="components-textinput--kitchen-sink" />
+</Canvas>
+
+### Label
+
+A text label for the Field. The associated element may be `<label>` or `<legend>` depending on Field's Input type.
+
+<Canvas>
+  <Story id="components-field--field-label" />
+</Canvas>
+
+The label also indicates the Field's optional/required status to ensure screen readers properly announce the state.
+
+<Canvas>
+  <Story id="components-field--field-optional" />
+</Canvas>
+
+### Hint
+
+A hint is helper text for the Field. The field hint is similar to the disallowed `placeholder` attribute. It should provide any necessary help text for the Field but is optional.
```

Comment:
disallowed placeholder attribute? if we don't want placeholders (which I wasn't aware of!) we should prevent their use in code too. Can you update the types for our input components to make this so before we merge the docs in this form? Alternatively, leave this line of docs off and we'll update everything in another subsequent PR at a later point.

Reply:
"disallowed" is definitely too strong here, as there are places that `placeholder` is an existing escape hatch. I'll update the language. 

As guidance, we want designers and developers to avoid `placeholder` as it comes with a11y and usability issues.